### PR TITLE
ACM#9626: Update the  command params

### PR DESCRIPTION
**Pull request by xenolinux**
This PR fixes the following Jiras:
- https://issues.redhat.com/browse/ACM-9625
- https://issues.redhat.com/browse/ACM-9626

This PR updates the parameters in the `hcp create nodepool agent` command.

### CODE REVIEW

File: clusters/hosted_control_planes/scale_nodepool_hosted_bm.adoc [hunk 1]
```diff
@@ -113,33 +113,33 @@ clusterversion.config.openshift.io/version             False       True
...
+clusteroperator.config.openshift.io/storage                                    4.12z    True        False         False      11m
 ----
 
 [#adding-nodepool-bm]
 == Adding node pools
 
 You can create node pools for a hosted cluster by specifying a name, number of replicas, and any additional information, such as an agent label selector.
 
-. To create a node pool, enter the following information. In this example, the node pool uses agents with the `"size" : "medium"` label:
+. To create a node pool, enter the following information. In this example, the node pool uses agents with the 'size=large,zone notin (az1,az2)' label:
```
Comment (dockerymick) [inline clusters/hosted_control_planes/scale_nodepool_hosted_bm.adoc]: use backticks to create monospace
```suggestion
. To create a node pool, enter the following information. In this example, the node pool uses agents with the `'size=large,zone notin (az1,az2)'` label:
```

**Review by dockerymick — CHANGES_REQUESTED** [review]
Please see my comment. Thanks for the updates!

**Comment by avishayt** [comment]
LGTM

File: clusters/hosted_control_planes/scale_nodepool_hosted_bm.adoc (hunk 1 above)
Comment (xenolinux) [reply clusters/hosted_control_planes/scale_nodepool_hosted_bm.adoc]: Addressed, ty

File: clusters/hosted_control_planes/scale_nodepool_hosted_bm.adoc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [#scaling-the-nodepool]
 = Scaling the NodePool object for a hosted cluster
 
-You add nodes to your hosted cluster by scaling the `NodePool` object.  
+You add nodes to your hosted cluster by scaling the `NodePool` object.
```
Comment (dockerymick) [inline clusters/hosted_control_planes/scale_nodepool_hosted_bm.adoc:4]:
```suggestion
To add nodes to your hosted cluster, scale the `NodePool` object. Complete the following steps:
```

**Review by dockerymick — APPROVED** [review]
I realize that i didn't submit my review. My apologies. I left 1 suggestion

File: clusters/hosted_control_planes/scale_nodepool_hosted_bm.adoc (hunk 2 above)
Comment (xenolinux) [reply clusters/hosted_control_planes/scale_nodepool_hosted_bm.adoc:4]: Suggestion is addressed in https://github.com/stolostron/rhacm-docs/pull/6139

**Pull request merged**